packages/perseus/src/components/fixed-to-responsive.tsx

```diff
@@ -70,13 +75,13 @@ const FixedToResponsive: any = createReactClass({
         } else {
             this._cacheViewportSize();
         }
-    },
+    }
 
-    componentWillUnmount: function () {
+    componentWillUnmount() {
         this._isMounted = false;
-    },
+    }
 
-    _cacheViewportSize: function () {
+    _cacheViewportSize: () => void = (): void => {
```

Comment:
```suggestion
    _cacheViewportSize(): void {
```
Would something like this work?

Reply:
Unfortunately, no.  We use this function in a `setTimeout()` and if we don't use an arrow function, the `this` binding is lost (regular instance functions don't retain their `this` binding like arrow functions do. :(